VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -460,22 +558,28 @@ public void run() {
     }
 
     /**
+     * callback from EventBus
```

Comment:
Seems like a useless javadoc, should either be filled with proper comments or deleted if the event is \*self-explanatory\*.